aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -112,6 +113,18 @@ Additional HTTP.sys configuration is handled through [registry settings](https:/
 
 For more information about HTTP.sys options, see <xref:Microsoft.AspNetCore.Server.HttpSys.HttpSysOptions>.
 
+### Customize security descriptors
+
+[A *request queue* in HTTP.sys](https://www.bing.com/search?q=all-about-iis-asp-net-request-queues) is a kernel-level structure that temporarily stores incoming HTTP requests until your application is ready to process them. Manage access to the request queue by using the [RequestQueueSecurityDescriptor](https://source.dot.net/#Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs,a556950881fd2d87) property on <xref:Microsoft.AspNetCore.Server.HttpSys.HttpSysOptions>. Set it to a <xref:System.Security.AccessControl.GenericSecurityDescriptor> instance when configuring your HTTP.sys server.
```

Comment:
Probably shouldn't link to a search engine query

Reply:
I'll delete that link.